test/integration/cases_test.go

```diff
@@ -192,6 +277,54 @@ type upnessMetrics struct {
 	down_time_between_recovery_latest_seconds   metric
 }
 
+type utilizationMetrics struct {
+	// Always present
```

Comment:
Is `job_scheduled` `//Always present`, I thought it only existed upon scheduling.

Reply:
@nstogner You're right, and that was a big source of confusion for me actually, because my thinking was that for this metric to be useful, it would need to exist but report a 0 value if there were nodepools that did not have jobs scheduled on them. As is, whenever the metric exists, the value will be 1, right?

Reply:
updated comment in any case